src/documents/ChangeApplierAPI.md

```diff
@@ -272,8 +395,13 @@ function listener(value, oldValue, pathSegs, changeRequests)
             <td>An array of <code>String</code> path segments holding the path at which <code>value</code> and <code>oldValue</code> are/were held</td>
         </tr>
         <tr>
-            <td><code>changeRequests</code></td>
-            <td>A single <code>ChangeRequest</code> object or an array of them (see below) which were responsible for this change (may be empty)</td>
+            <td><code>changeRequest</code></td>
+            <td>May contain a single <code>ChangeRequest</code> object which was responsible for this change, but will often be empty. This signature element is not a stable API</td>
```

Comment:
It should not be used, in general. Sometimes there is no single ChangeRequest object responsible for this change.